config/default/core.entity_view_display.node.localgov_news_article.related_anonymous.yml

```diff
@@ -0,0 +1,93 @@
+uuid: 94ed03c8-b1ff-493d-bfae-c7b85edbc8fe
+langcode: en
+status: true
+dependencies:
+  config:
+    - core.entity_view_mode.node.related_anonymous
+    - facets.facet.localgov_news_category
+    - field.field.node.localgov_news_article.body
+    - field.field.node.localgov_news_article.field_content_owner
+    - field.field.node.localgov_news_article.field_content_sme
+    - field.field.node.localgov_news_article.field_media_image
+    - field.field.node.localgov_news_article.localgov_news_categories
+    - field.field.node.localgov_news_article.localgov_news_date
+    - field.field.node.localgov_news_article.localgov_news_related
+    - field.field.node.localgov_news_article.localgov_newsroom
+    - field.field.node.localgov_news_article.localgov_restricted_content
+    - node.type.localgov_news_article
+  module:
+    - datetime
+    - entity_reference_facet_link
+    - media
+    - text
+    - user
+  enforced:
+    module:
+      - localgov_news
+_core:
+  default_config_hash: T3PrBFQlZHLL6PIlyZqMvmSjh3fs5EiJsyXHC_Z1uMw
+id: node.localgov_news_article.related_anonymous
+targetEntityType: node
+bundle: localgov_news_article
+mode: related_anonymous
+content:
+  body:
```

Comment:
Body, content_moderation_control, links, localgov_news_categories should be hidden